### ISSUE TITLE
{"status":3017,"msg":"0 is less than the min limit order price 0.1"} - FCOIN

### ISSUE THREAD

**Issue by bestlinkXx** [ccxt/ccxt#5952]
Hello, 

I want to do a create.order() with fok in type_order on FCOIN.
But I have this kind of error : {"status":3017,"msg":"0 is less than the min limit order price 0.1"}

I don't understand how I can fix it, perhaps add something in parameters.
I checked on FCOIN API DOC and I found nothing.

Method : result = self.client.create_order(symbol=symbol, type="fok", side="buy", amount=balance,price=ask_price)
Can you please help me ?

Regards

**Comment by kroitor** [comment]
@bestlinkXx hi! Can you please post the complete but short snippet of code 10-20 lines to reproduce it, including the exchange instantiation, the symbol used and other params, but without your keys? We also require your verbose request/response to get this resolved quickly, please read these guidelines carefully:

- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-submit-an-issue
- https://github.com/ccxt/ccxt/wiki/FAQ#i-am-calling-a-method-and-i-get-an-error-what-am-i-doing-wrong

**Comment by bestlinkXx** [comment]
Hi Igor,

I'll try to do my best.
```
client = ccxt.fcoin({
        'apiKey': "MyApiKey",
        'secret': "MyApiSecret",
})
result = self.client.create_order(symbol="ETH/USDT", type="fok", side="buy", amount=1,price=200)
```

```
WARNING: ETH/USDT: fcoin {"status":3017,"msg":"0 is less than the min limit order price 0.01"}
```

I hope I put all information that you need.
So I'm trying to find a solution to put FOK or IOC order on Fcoin.
Thank you Igor !

**Comment by kroitor** [comment]
@bestlinkXx this is fixed in CCXT version 1.18.1246, and it will be available for downloading in 15 minutes. Thx for reporting!
